### PR TITLE
Add call depth level to debugging messages

### PR DESCRIPTION
**Pull request by axic**
Split off #425.

### CODE REVIEW

**Comment by axic** [comment]
@chfast can you review please?

File: src/eei.h [hunk 1]
```diff
@@ -136,6 +136,9 @@ class EthereumInterface {
   void eeiRevertOrFinish(bool revert, uint32_t offset, uint32_t size);
 
   // Helpers methods
+  inline std::string depthToString() const {
+    return std::string("[") + std::to_string(m_msg.depth) + "]";
```
Comment (chfast) [inline src/eei.h]:
```suggestion
    return "[" + std::to_string(m_msg.depth) + "]";
```

Comment (axic) [reply src/eei.h]: I think it wasn't compiling hence the explicit `string("[")`

Comment (axic) [reply src/eei.h]: Changed.

**Pull request merged**